### PR TITLE
Adds Issue Template for `bug`, `feature`, `config`

### PR DESCRIPTION
**Pull request by PSGitHubUser1**
Same as Title.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      ...
+   validations:
+     required: true
+ - type: dropdown
+   attributes:
+    label: Operating system
+    description: Which operating system do you use?
+    options:
+     - Windows
+     - Linux
```
Comment (Goldorion) [inline .github/ISSUE_TEMPLATE/bug_report.yml:29]: Add macOS

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+     - Other (Tell us at the Specifications part)
```
Comment (Goldorion) [inline .github/ISSUE_TEMPLATE/bug_report.yml:30]: This is not needed as there are only 3 OS supported by Minecraft and they are Windows, macOS and Linux, the only 3 OS used for computers.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    placeholder: |
+      Version: [v1.2 / v2.1 ...]
+      
+   validations:
+     required: true
+ - type: textarea
+   attributes:
+    label: Example workspace
+    description: |
+      If the bug is related to the workspace, always provide a minimal reproducible example workspace, export it to a shareable ZIP file, and attach it to the ticket.
```
Comment (Goldorion) [inline .github/ISSUE_TEMPLATE/bug_report.yml:46]: This is a mod... not MCreator.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    label: Feature description
+    description: Explain clearly
+    placeholder: |
+     Tell us here about the feature
+   validations:
+    required: true
+ - type: textarea
+   attributes:
+    label: Logs
+    description: Provide loags [Optional]
```
Comment (Goldorion) [inline .github/ISSUE_TEMPLATE/feature_request.yml:16]: loags?

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+name: Feature Request
+description: Create a report to help us improve (search for duplicates first)
```
Comment (Goldorion) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: Change for something like Request a feature

**Review by Goldorion — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (PSGitHubUser1) [reply .github/ISSUE_TEMPLATE/bug_report.yml:29]: Okey.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (PSGitHubUser1) [reply .github/ISSUE_TEMPLATE/bug_report.yml:30]: Oh

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (PSGitHubUser1) [reply .github/ISSUE_TEMPLATE/bug_report.yml:46]: OHHH YEA!

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 4 above)
Comment (PSGitHubUser1) [reply .github/ISSUE_TEMPLATE/feature_request.yml:16]: Thanks.

**Comment by PSGitHubUser1** [comment]
Done!!

**Pull request closed without merging**